packages/experimental/Badge/src/PresenceBadge/presenceIconPaths.ts

```diff
@@ -2,21 +2,21 @@ export const presenceIconPaths = {
-  outOfOfficeHC: `<path d="M9.06842 6.20736C9.45895 5.81684 9.45896 5.18368 9.06845 4.79315C8.67793 4.40262 8.04476 4.40261 7.65423 4.79313L5.15423 7.29304C4.96669 7.48058 4.86133 7.73493 4.86133 8.00015C4.86133 8.26537 4.96668 8.51973 5.15422 8.70727L7.65422 11.2073C8.04475 11.5978 8.67791 11.5978 9.06843 11.2073C9.45896 10.8167 9.45896 10.1836 9.06843 9.79305L8.27554 9.00016H11.8613C12.4136 9.00016 12.8613 8.55244 12.8613 8.00016C12.8613 7.44788 12.4136 7.00016 11.8613 7.00016H8.2756L9.06842 6.20736ZM8.86133 0C4.44305 0 0.861328 3.58172 0.861328 8C0.861328 12.4183 4.44305 16 8.86133 16C13.2796 16 16.8613 12.4183 16.8613 8C16.8613 3.58172 13.2796 0 8.86133 0ZM2.86133 8C2.86133 4.68629 5.54762 2 8.86133 2C12.175 2 14.8613 4.68629 14.8613 8C14.8613 11.3137 12.175 14 8.86133 14C5.54762 14 2.86133 11.3137 2.86133 8Z" fill="white"/>`,
-  availableOutOfOfficeHC: `<path d="M12.5684 6.70711C12.959 6.31658 12.959 5.68342 12.5684 5.29289C12.1779 4.90237 11.5447 4.90237 11.1542 5.29289L7.86133 8.58579L6.56843 7.29289C6.17791 6.90237 5.54475 6.90237 5.15422 7.29289C4.7637 7.68342 4.7637 8.31658 5.15422 8.70711L7.15422 10.7071C7.54475 11.0976 8.17791 11.0976 8.56843 10.7071L12.5684 6.70711ZM0.861328 8C0.861328 3.58172 4.44305 0 8.86133 0C13.2796 0 16.8613 3.58172 16.8613 8C16.8613 12.4183 13.2796 16 8.86133 16C4.44305 16 0.861328 12.4183 0.861328 8ZM8.86133 2C5.54762 2 2.86133 4.68629 2.86133 8C2.86133 11.3137 5.54762 14 8.86133 14C12.175 14 14.8613 11.3137 14.8613 8C14.8613 4.68629 12.175 2 8.86133 2Z" fill="white"/>`,
-  doNotDisturbOutOfOfficeHC: `<path d="M6.11133 7C5.55904 7 5.11133 7.44772 5.11133 8C5.11133 8.55228 5.55904 9 6.11133 9H11.6113C12.1636 9 12.6113 8.55228 12.6113 8C12.6113 7.44772 12.1636 7 11.6113 7H6.11133ZM0.861328 8C0.861328 3.58172 4.44305 0 8.86133 0C13.2796 0 16.8613 3.58172 16.8613 8C16.8613 12.4183 13.2796 16 8.86133 16C4.44305 16 0.861328 12.4183 0.861328 8ZM8.86133 2C5.54762 2 2.86133 4.68629 2.86133 8C2.86133 11.3137 5.54762 14 8.86133 14C12.175 14 14.8613 11.3137 14.8613 8C14.8613 4.68629 12.175 2 8.86133 2Z" fill="white"/>`,
+  availableHC: `<path d="M8 16A8 8 0 108 0a8 8 0 000 16zm3.7-9.3l-4 4a1 1 0 01-1.41 0l-2-2a1 1 0 111.42-1.4L7 8.58l3.3-3.3a1 1 0 011.4 1.42z" fill="white"/>`,
```

Comment:
Is white always the proper fill color for HC? Or should the fill color depend on a specific brush/color token?

Reply:
Honestly, I don't know. In another branch there can be any color. White is by design. But I think it would be good to have color depending on theme